vng_api_common/notifications/models.py

```diff
@@ -91,7 +85,18 @@ def register(self) -> None:
         }
 
         # register the subscriber
-        subscriber = client.create("abonnement", data=data)
-
-        self._subscription = subscriber["url"]
+        try:
+            response = client.post("abonnement", data=data)
+            data = to_internal_data(response)
+        except RequestException:
+            raise RuntimeError(f"Failed adding subscription for {self.callback_url}")
+        except ClientError as exc:
+            response_json = exc.args[0]
+
+            raise RuntimeError(
```

Comment:
Not sure what errors were possible previously, but I don't think `RuntimeError` was one of them? I think I'd just log and raise the `RequestException`